### PR TITLE
remote client

### PR DESCRIPTION
**Pull request by spetrychenko**

### CODE REVIEW

File: xs2a-adapter-service-remote/src/main/java/de/adorsys/xs2a/adapter/service/impl/AccountInformationServiceImpl.java [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+    @Override
+    public GeneralResponse<ConsentCreationResponse> createConsent(RequestHeaders requestHeaders, Consents consents) {
+        ConsentsTO consentsTO = consentMapper.toConsentsTO(consents);
+        ResponseEntity<ConsentsResponse201TO> responseEntity = client.createConsent(requestHeaders.toMap(), consentsTO);
+        ConsentCreationResponse consentCreationResponse = creationResponseMapper.toConsentCreationResponse(responseEntity.getBody());
+        return new GeneralResponse<>(responseEntity.getStatusCodeValue(), consentCreationResponse, getHeaders(responseEntity.getHeaders()));
+    }
+
+    @Override
+    public GeneralResponse<ConsentInformation> getConsentInformation(String s, RequestHeaders requestHeaders) {
```
Comment (n0rdy) [inline xs2a-adapter-service-remote/src/main/java/de/adorsys/xs2a/adapter/service/impl/AccountInformationServiceImpl.java]: Maybe we should rename `String s` to something more readable, what do you think?
The same story with the rest of the `String s` within this class.

File: xs2a-adapter-service-remote/src/main/java/de/adorsys/xs2a/adapter/service/impl/AccountInformationServiceImpl.java [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+            if (value != null && !value.isEmpty()) {
+                headers.put(entry.getKey(), value.get(0));
+            }
+        }
+        return ResponseHeaders.fromMap(headers);
+    }
+
+    private LocalDate strToLocalDate(String date) {
+        if (StringUtils.isNotBlank(date)) {
+            return LocalDate.parse(date);
```
Comment (n0rdy) [inline xs2a-adapter-service-remote/src/main/java/de/adorsys/xs2a/adapter/service/impl/AccountInformationServiceImpl.java]: Is it a correct behaviour for us: if `String date` has incorrect format -> `LocalDate.parse(date)` throws `DateTimeParseException` -> user gets `500` response?

File: xs2a-adapter-service-remote/src/main/java/de/adorsys/xs2a/adapter/service/impl/PaymentInitiationServiceImpl.java [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+    private final PaymentInitiationAuthorisationResponseMapper authorisationResponseMapper = Mappers.getMapper(PaymentInitiationAuthorisationResponseMapper.class);
+    private final StartScaProcessResponseMapper scaProcessResponseMapper = Mappers.getMapper(StartScaProcessResponseMapper.class);
+    private final ObjectMapper objectMapper = new ObjectMapper();
+
+    public PaymentInitiationServiceImpl(PaymentInitiationClient paymentInitiationClient) {
+        this.client = paymentInitiationClient;
+    }
+
+    @Override
+    public GeneralResponse<PaymentInitiationRequestResponse> initiateSinglePayment(String s, RequestHeaders requestHeaders, Object o) {
```
Comment (n0rdy) [inline xs2a-adapter-service-remote/src/main/java/de/adorsys/xs2a/adapter/service/impl/PaymentInitiationServiceImpl.java]: The same question about `String s` as above

File: xs2a-adapter-service-remote/src/main/java/de/adorsys/xs2a/adapter/service/impl/AccountInformationServiceImpl.java (hunk 2 above)
Comment (spetrychenko) [reply xs2a-adapter-service-remote/src/main/java/de/adorsys/xs2a/adapter/service/impl/AccountInformationServiceImpl.java]: added parse exception and handler in the example app

**Pull request merged**